devtasks.py

```diff
@@ -78,9 +79,10 @@ def lint():
             )
         except ProcessExecutionError:
             _logger.info(
-                "Couldn't create copier-lint-v1 container, probably because a previous one exists. "
-                "Remove it if you want to recycle it. Otherwise, this is OK."
+                "Couldn't create copier-lint-v1 container, probably because a previous "
+                "one exists. Remove it if you want to recycle it. Otherwise, this is"
+                "OK."
```

Comment:
I thought the strings were split automatically by Ruff, but I see here that a space is missing at the end of `is` or at the beginning of `OK`, so I suppose you split them manually?

Reply:
i split them manually. i think black --preview can automatically re-split strings?